package.json

```diff
@@ -1,9 +1,15 @@
 {
   "name": "kizuna-platform",
-  "version": "1.0.0",
+  "version": "0.0.1",
   "description": "Friendship Game.  For Vote, Rank,  QA ...",
   "main": "index.js",
   "scripts": {
+    "build": "npm run clean && webpack --config ./webpack.prod.config.js",
+    "clean": "rm -rf build",
+    "eslint": "eslint src test",
+    "lint": "npm run eslint && npm run stylelint",
+    "start": "webpack-dev-server --devtool eval --progress --colors --hot --history-api-fallback --content-base dist",
+    "stylelint": "stylelint \"src/**/*.scss\" --syntax scss",
```

Comment:
Never mind. according to [run-script | npm Documentation](https://docs.npmjs.com/cli/run-script), what you have here is a best practice:

> For example, if there is a devDependency on tap in your package, you should write:
> ```
> "scripts": {"test": "tap test/\*.js"}
> ```
> instead of "scripts": {"test": "node_modules/.bin/tap test/\*.js"} to run your tests.